tests/testthat/test-slice.R

```diff
@@ -1,9 +1,12 @@
-context("test-slice")
```

Comment:
Why this change?

Reply:
`context()` is no longer needed in recent testthat.

Reply:
Looks like context are now generated from the file names, cool.